README.md

```diff
@@ -2,73 +2,64 @@
 
 This is a sandbox project for exploring the basic functionality and latest features of dbt. It's based on a fictional restaurant called the Jaffle Shop that serves [jaffles](https://en.wikipedia.org/wiki/Pie_iron). Enjoy!
 
-[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/dbt-labs/jaffle-shop?quickstart=1)  
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/dbt-labs/jaffle-shop)
-
 ## Create new repo from template
 
 1. <details>
    <summary>Click the green "Use this template" button at the top of the page to create a new repository from this template.</summary>
 
    ![Click 'Use this template'](/.github/static/use-template.gif)
    </details>
-3. Follow the steps to create a new repository.
+
+2. Follow the steps to create a new repository.
 
 ## Platform setup
 
 ### dbt Cloud IDE (most beginner friendly)
+
 1. Set up a dbt Cloud account and follow Step 4 in the [Quickstart instructions for your data platform](https://docs.getdbt.com/quickstarts), to connect your platform to dbt Cloud.
+
 2. Choose the repo you created in Step 1 as the repository for your dbt Project code.
+
 3. Click `Develop` in the top nav, you should be prompted to run a `dbt deps`, which you should do.
 
-### dbt Cloud in GitHub Codespaces (more customizable)
+### dbt Cloud CLI (if you prefer to work locally)
```

Comment:
Is it worth calling out that you need to link this repo to a cloud project as a part of setup?

Reply:
it's in the instructions as a link, felt like it was better to kick out to the CLI install page for those instructions instead of complicating the README (for now, until slurp time)